### PR TITLE
Add support for TAINT_SUPPORT_REMOVED (#1601545)

### PR DESCRIPTION
**Pull request by poncovka**
Show a message if the installer detects unsupported hardware.

Resolves: rhbz#1601545

### CODE REVIEW

**Comment by poncovka** [comment]
screenshot from 2018-09-14 14-19-58

**Comment by poncovka** [comment]
screenshot from 2018-09-14 17-09-31

File: pyanaconda/core/util.py [hunk 1]
```diff
@@ -959,21 +960,74 @@ def conditional_chown(path, uid, gid, from_uid=None, from_gid=None):
...
+    :param install_class: an install class
+    :return: a list of warnings
+    """
+    warnings = []
+
+    if flags.automatedInstall or flags.dirInstall or flags.imageInstall:
+        log.info("Skipping detection of unsupported hardware.")
+        return []
+
+    # Check TAINT_HARDWARE_UNSUPPORTED.
```
Comment (jkonecny12) [inline pyanaconda/core/util.py]: Remove the period on the end of comment.

File: pyanaconda/core/util.py [hunk 2]
```diff
@@ -959,21 +960,74 @@ def conditional_chown(path, uid, gid, from_uid=None, from_gid=None):
...
+    if not install_class.detect_unsupported_hardware:
+        log.debug("Skipping detection of TAINT_HARDWARE_UNSUPPORTED.")
+
+    elif get_kernel_taint(TAINT_HARDWARE_UNSUPPORTED):
```
Comment (jkonecny12) [inline pyanaconda/core/util.py]: Remove blank line. You don't need them especially when there is if with just one line.

File: pyanaconda/core/util.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/core/util.py]: Fixed.

File: pyanaconda/core/util.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/core/util.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**